src/pages/SingleArticle.jsx

```diff
@@ -13,6 +13,20 @@ function SingleArticle() {
     });
   }, [article_id]);
 
+  function handleLike(event) {
+    event.preventDefault();
+    patchArticleVotes(article_id, 1).then((updatedArticle) => {
+      setArticle(updatedArticle);
+    });
+  }
+
+  function handleDislike(event) {
+    event.preventDefault();
+    patchArticleVotes(article_id, -1).then((updatedArticle) => {
+      setArticle(updatedArticle);
+    });
+  }
+
```

Comment:
You need to handle these optimistically.
There some good notes on it. 

At the moment, its really slow is my internet is slow, and could result in my clicking the button lots of times, thinking it isnt working, but its just being slow on the api request.